config/postcss.config.js

```diff
@@ -33,8 +34,7 @@ module.exports = {
             level: 1
         }),
         postcssCustomProperties({
-            preserve: true,
-            importFrom: 'dist/root.css' //replace with theme-designer variables
```

Comment:
There is no need to import the file again.  The fallback values for the CSS variables (from the `postcssAddFallback` plugin) will be used to generate the custom properties.